lib/ansible/modules/windows/win_domain_user.py

```diff
@@ -75,11 +75,9 @@
     description:
       - C(always) will update passwords if they differ.
```

Comment:
Looks like there is a mistake in the docs, are you able to change it to something like `C(always) will always update the password`.